examples/comet/comet-pytorch.ipynb

```diff
@@ -0,0 +1,439 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "b8954812",
+   "metadata": {},
+   "source": [
+    "# Track and Compare Models with Comet"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "11e1e3e6",
+   "metadata": {},
+   "source": [
+    "![Comet Logo](https://saturn-public-assets.s3.us-east-2.amazonaws.com/example-resources/comet-logo.png \"doc-image\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Overview\n",
+    "\n",
+    "[Comet](https://www.comet.ml/) is a platform for tracking, comparing, and visualizing your modeling workflows. This example shows how to use Comet on the Saturn Cloud platform by creating a PyTorch classification model of the MNIST digits dataset, but Comet can be used to log many types of model training. Check out the rest of the [Comet documentation](https://www.comet.ml/docs/) for examples of how to integrate Comet with other frameworks and languages. \n",
+    "\n",
+    "## Process\n",
+    "\n",
+    "### Find your Comet API key and workspace name\n",
+    "\n",
+    "Follow the [Comet instructions](https://www.comet.ml/docs/user-interface/) to find the workspace name where you want to log this modeling run and the corresponding API key.\n",
+    "\n",
+    "### Add your Comet API key to Saturn Cloud\n",
```

Comment:
This is all gonna change in line a week when we switch from credentials to secrets. I think we should replace this with a like:

> You can specify the API key within your notebook by using `experiment = comet_ml.Experiment("YOUR-API-KEY")` [or whatever]. Alternatively, you can more securely manage that API key and use it across multiple Saturn Cloud resources by storing it within Saturn Cloud [secrets manager](link) using the environment variable name `COMET_API_KEY`.

For now that link would just link to the credentials docs but in the future we'll make the credentials docs automatically redirect to the new secrets docs